resource/ui/ts/typings/mithriljs/mithril.d.ts

```diff
@@ -122,8 +122,8 @@ declare module _mithril {
 		toJSON(): T;
 	}
 
-	interface MithrilRoutes<T extends MithrilController> {
```

Comment:
The typescript typings were incorrect.  Matt was the one who actually did the work on this.  If we don't do this, the transpiler fails when it tries to sort out the routing table in app.ts.